### ISSUE TITLE
improve clear.sh to make it accessible on production env

### ISSUE THREAD

**Issue by vdelendik** [qaprosoft/qps-infra#136]
1. it should explicitly say that all stored data (jenkins, elasticsearch, sonarqube etc) will be permanently removed. Only after confirmation continue to step #2
2. verify if we already have backups for the configured components. #135 is related ticket about backup. 
a) If no backup detected inform user one more time that no backup and all data will be permanently removed.
b) if backup(s) exist inform about date when backup(s) was/were generated
3. only after confirmation on step #2 start actual clearing

**Comment by vdelendik** [comment]
won't fix. huge refactoring done to have backup/restore/shutdown etc functionality.
